<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Subscriber</class>
 <widget class="QMainWindow" name="Subscriber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>隶书</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>华文隶书</family>
           <pointsize>36</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;华文隶书&quot;;
color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>从控机系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="on">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>电源</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exit">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 127);</string>
           </property>
           <property name="text">
            <string>退房</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="set_temp">
           <property name="font">
            <font>
             <family>华文新魏</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>温度+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_temp_2">
           <property name="font">
            <font>
             <family>华文新魏</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>温度-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_wind">
           <property name="font">
            <font>
             <family>华文新魏</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 0);</string>
           </property>
           <property name="text">
            <string>风速</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_mode">
           <property name="font">
            <font>
             <family>华文新魏</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 255, 0);</string>
           </property>
           <property name="text">
            <string>模式</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>费  用：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>华文琥珀</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>设定风速：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>华文琥珀</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>当前风速：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="model_of_main">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="cost">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="wind_speed_2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>华文琥珀</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>当前温度：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="wind_speed">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>华文琥珀</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>空调模式：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcd_goaltemp"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>华文琥珀</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>目标温度：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLCDNumber" name="lcd_currenttemp"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>on</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>state_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_mode</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>mode_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_wind</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>speed_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_temp</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>temperature_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_temp_2</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>temperature_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Subscriber</receiver>
   <slot>check_out()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>state_change()</slot>
  <slot>mode_change()</slot>
  <slot>speed_change()</slot>
  <slot>temperature_up()</slot>
  <slot>temperature_down()</slot>
  <slot>check_out()</slot>
 </slots>
</ui>
